### PR TITLE
Starting with minor optimization and reduce #lines of code

### PR DESCRIPTION
**Pull request by rahulchaudharyofficial**
Starting with minor changes, please review.

---
This change is [Reviewable](https://reviewable.io/reviews/apex-enterprise-patterns/fflib-apex-common/328)

### CODE REVIEW

**Review by wimvelzeboer — CHANGES_REQUESTED** [review]

**Comment by ImJohnMDaniel** [comment]
@rahulchaudharyofficial can you first discuss more about what your intentions are with this PR please?  Is there an issue that this trying to solve?  Why should this change be needed?  Thanks for the help.

File: sfdx-source/apex-common/main/classes/fflib_SObjectDescribe.cls [hunk 1]
```diff
@@ -263,28 +263,29 @@ public class fflib_SObjectDescribe {
...
-				return null;
-			String preferredValue = ((implyNamespace ? currentNamespace+'__' : '') + name).toLowerCase();
-			if(values.containsKey(preferredValue)){
-				return values.get(preferredValue);
-			}else if(implyNamespace){
-				return values.get(name.toLowerCase());
-			}else{
-				return null;
+			Object toReturn = null;
+			if(!String.isBlank(name))
```
[the remark this answers is not in the document]
Comment (rahulchaudharyofficial) [reply sfdx-source/apex-common/main/classes/fflib_SObjectDescribe.cls:267]: Thank you for the feedback.
my intention is to not compare strings directly with null and let String class's isBlank method deal with it's null or blank possibilities.
I do agree that Negative or Positive are both ways of writing the block but readability in this non complex scenario is not a huge factor.
Existing code was nested with multiple if clauses and i am trying to reduce it.

**Comment by rahulchaudharyofficial** [comment]
@ImJohnMDaniel My intensions are following

a) I am trying to remove explicit null checks against a string and use String's build in API which checks for null as well as Blank string.
b) I am trying to avoid multiple nesting of block.
c) From functional point of view this PR do not change anything but idea here is to make it more compact and concise.

**Comment by daveespo** [comment]
We're going to reject this PR -- while we appreciate the need to refactor, we would like that coupled with a functional need for the refactoring (i.e. fixing a bug or adding new functionality)

Just moving around code for readability creates unnecessary risk.

**Pull request closed without merging**